app/database_etl/location_utils/country_iso2.json

```diff
@@ -1,59 +1,241 @@
+  "Latvia": "LV",
+  "Lebanon": "LB",
+  "Lesotho": "LS",
+  "Liberia": "LR",
+  "Libya": "LY",
+  "Liechtenstein": "LI",
+  "Lithuania": "LT",
+  "Luxembourg": "LU",
+  "Madagascar": "MG",
+  "Malawi": "MW",
+  "Malaysia": "MY",
+  "Maldives": "MV",
+  "Mali": "ML",
+  "Malta": "MT",
+  "Marshall Islands": "MH",
+  "Martinique": "MQ",
+  "Mauritania": "MR",
+  "Mauritius": "MU",
+  "Mayotte": "YT",
+  "Mexico": "MX",
+  "Micronesia (Federated States of)": "FM",
+  "Monaco": "MC",
+  "Mongolia": "MN",
+  "Montenegro": "ME",
+  "Montserrat": "MS",
+  "Morocco": "MA",
+  "Mozambique": "MZ",
+  "Myanmar": "MM",
+  "Namibia": "NA",
+  "Nauru": "NR",
+  "Nepal": "NP",
+  "Netherlands": "NL",
+  "New Caledonia": "NC",
+  "New Zealand": "NZ",
+  "Nicaragua": "NI",
+  "Niger": "NE",
+  "Nigeria": "NG",
+  "Niue": "NU",
+  "Norfolk Island": "NF",
+  "North Macedonia": "MK",
+  "Northern Mariana Islands (Commonwealth of the)": "MP",
+  "Norway": "NO",
+  "occupied Palestinian territory, including east Jerusalem": "PS",
+  "Oman": "OM",
+  "Pakistan": "PK",
+  "Palau": "PW",
+  "Panama": "PA",
+  "Papua New Guinea": "PG",
+  "Paraguay": "PY",
+  "Peru": "PE",
+  "Philippines": "PH",
+  "Pitcairn Islands": "PN",
+  "Poland": "PL",
+  "Portugal": "PT",
+  "Puerto Rico": "PR",
+  "Qatar": "QA",
+  "Republic of Korea": "KR",
+  "Republic of Moldova": "MD",
+  "Romania": "RO",
+  "Russian Federation": "RU",
+  "Rwanda": "RW",
+  "R\u00e9union": "RE",
+  "Saint Barth\u00e9lemy": "BL",
+  "Saint Helena": "SH",
+  "Saint Kitts and Nevis": "KN",
+  "Saint Lucia": "LC",
+  "Saint Martin": "MF",
+  "Saint Pierre and Miquelon": "PM",
+  "Saint Vincent and the Grenadines": "VC",
+  "Samoa": "WS",
+  "San Marino": "SM",
+  "Sao Tome and Principe": "ST",
+  "Saudi Arabia": "SA",
+  "Senegal": "SN",
+  "Serbia": "RS",
+  "Seychelles": "SC",
+  "Sierra Leone": "SL",
+  "Singapore": "SG",
+  "Sint Maarten": "SX",
+  "Slovakia": "SK",
+  "Slovenia": "SI",
+  "Solomon Islands": "SB",
+  "Somalia": "SO",
+  "South Africa": "ZA",
+  "South Sudan": "SS",
+  "Spain": "ES",
+  "Sri Lanka": "LK",
+  "Sudan": "SD",
+  "Suriname": "SR",
+  "Svalbard and Jan Mayen Islands": "SJ",
+  "Sweden": "SE",
+  "Switzerland": "CH",
+  "Syrian Arab Republic": "SY",
+  "Tajikistan": "TJ",
+  "Thailand": "TH",
+  "The United Kingdom": "GB",
+  "Timor-Leste": "TL",
+  "Togo": "TG",
+  "Tokelau": "TK",
+  "Tonga": "TO",
+  "Trinidad and Tobago": "TT",
+  "Tunisia": "TN",
+  "Turkey": "TR",
+  "Turkmenistan": "TM",
+  "Turks and Caicos Islands": "TC",
+  "Tuvalu": "TV",
+  "Uganda": "UG",
+  "Ukraine": "UA",
+  "United Arab Emirates": "AE",
+  "United Republic of Tanzania": "TZ",
+  "United States Virgin Islands": "VI",
+  "United States of America": "US",
+  "Uruguay": "UY",
+  "Uzbekistan": "UZ",
+  "Vanuatu": "VU",
+  "Venezuela (Bolivarian Republic of)": "VE",
+  "Viet Nam": "VN",
+  "Wallis and Futuna": "WF",
+  "Western Sahara": "EH",
+  "Yemen": "YE",
+  "Zambia": "ZM",
+  "Zimbabwe": "ZW"
 }
```

Comment:
You may notice that there are three less countries compared to `country_iso3.json`. The missing countries are: _Wake Island_, _Midway Islands_, and _Johnston Atoll_.  
  
This is because the codes for these countries have since been deleted (see [ISO 3166-1 alpha-3 Wiki](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3#Deleted_codes)). The ISO2 equivalents are different, and available at this [Wiki page](https://en.wikipedia.org/wiki/ISO_3166-2:UM).  

How should we navigate this? @atatmaja

Reply:
Apparently Wake island, Midway Islands, and johnston atoll are now part of the "United States Minor Outlying Islands", which has an iso code of UMI/UM. 

https://www.iso.org/obp/ui/#iso:code:3166:UM

Reply:
Yes – the second Wiki page I sent summarizes this.  
My question is: should we keep the old encoding to stay consistent with the ISO3 mapping we have, or should we use the new encoding here?

Reply:
I say we can just map all three to UMI/UM

Reply:
I think we can push this for now - it's not likely to affect us for a while (don't see us getting a study from any of these places anytime soon). 

Sometime soon, we can follow up with the WHO about this and see if they can revise their list of country name to code mappings. Does this sound like a plan @rahularoradfs

Reply:
Let's not sweat it too much - can follow up with Kathleen at WHO - will forward you a note that you can build off of to follow up if desired!